<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperacionesCampos</class>
 <widget class="QDialog" name="OperacionesCampos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="rb_decimal">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Decimal</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">fileType</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb_char">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>100</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Char</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">fileType</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb_noEsLLave">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>194</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>No es llave</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">llaves</string>
   </attribute>
  </widget>
  <widget class="QSpinBox" name="sp_decimales">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>48</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nameCampo">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>60</y>
     <width>220</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_primaria">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LLave primaria</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">llaves</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Decimales</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitud del campo</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sp_longitud">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>48</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del campo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_int">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>100</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Int</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">fileType</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb_ID">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">fileType</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>94</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de campo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_secundaria">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LLave secundaria</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">llaves</string>
   </attribute>
  </widget>
  <widget class="QComboBox" name="cb_campos">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>10</y>
     <width>371</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_eliminarCampo">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>270</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar Campo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modificarCampo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>225</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar Campo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Campos :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="llaves"/>
  <buttongroup name="fileType"/>
 </buttongroups>
</ui>
